src/__test__/Header.test.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('Header', () => {
+  it('renders the correct header content', () => {
+    render(
+      <BrowserRouter>
+        <Header />
+      </BrowserRouter>,
+    );
+
+    expect(screen.getByText(/sprintplex/i)).toBeInTheDocument();
+    expect(screen.getByText(/14:02/i)).toBeInTheDocument();
+    expect(screen.getByText(/100%/i)).toBeInTheDocument();
+    expect(
+      screen.getByText(/details of american companies/i),
+    ).toBeInTheDocument();
+    expect(screen.getByRole('link', { name: /details/i })).toHaveAttribute(
+      'href',
+      '/details',
+    );
```

Comment:
- [ ] While I run your tests, the following case failed .so kindly fix it and make sure all tests passed
>![image](https://user-images.githubusercontent.com/53402600/233785117-d60ae17b-6bfc-45df-9295-d6d80aab9d3e.png)